documentation/modules/ROOT/pages/operations/debezium-server.adoc

```diff
@@ -811,6 +811,25 @@ This value will be doubled upon every retry but won't exceed `debezium.sink.redi
 |`2000`
 |Socket timeout for Redis client.
 
+|[[redis-message-format]]<<redis-message-format, `debezium.sink.redis.message.format`>>
+|`extended`
+|The format of the message sent to the Redis stream. Possible values are `extended`(newer format) and `compact`(the until now old format). The `extended` format looks in Redis like this:
+[source]
+----
+1) 1) "1639304527499-0"
```

Comment:
Could you please make the format difference a bit more clear? Now it is hard to imagine.

Reply:
@jpechane could you give a hint, an example?

Reply:
@ggaborg
1) I'd recommend to add a link to the Redis Stream format
2) Larger snippet is needed, now it is not clear what is the difference between the two *schema* messages.

All in all it would probably be better to provide a separate section in the docs on the message format (and link) instead of embedding the short description in the table